task02_iryna/src/dao/Paragraph.java

```diff
@@ -0,0 +1,36 @@
+package dao;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Paragraph implements CompositeWord{
+
+    public static int sentensesCounter;
+
+    private List<LeafWord> sentenses;
+
+    public Paragraph(){
+        this.sentenses = new ArrayList<>();
+    }
+
+    @Override
+    public void add(LeafWord leaf) {
+        sentenses.add(leaf);
+    }
+
+    @Override
+    public String getData() {
+        StringBuilder res = new StringBuilder();
+        for (LeafWord sentence : sentenses) {
+            res.append(sentence.getData().trim()).append(". ");
+            sentensesCounter++;
```

Comment:
Correctly updating a static field from a non-static method is tricky to get right and could easily lead to bugs if there are multiple class instances and/or multiple threads in play. Ideally, static fields are only updated from synchronized static methods.